### PR TITLE
Update Developer Guide to include Help Command

### PR DESCRIPTION
**Pull request by ejnan**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -208,6 +212,40 @@ Tags: SevereAllergies
...
+Delete: delete INDEX
+   Example: delete 3
+
+Edit: edit INDEX [n\NAME] [ic\IC_NUMBER] [dob\DATE_OF_BIRTH] [ad\ADMISSION_DATE] [w\WARD] [t\TAG]...  
+   Example: edit 1 ic\T0123456P t\
+
+Find: find KEYWORD [MORE_KEYWORDS]
+   Example: find John 
+
+List: list
```
Comment (ryanlimdx) [inline docs/UserGuide.md:241]: Perhaps can include [t\TAG_KEYWORDS],  [w\WARD_KEYWORD] for list as well!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -265,3 +303,10 @@ Action | Format, Examples
 **Edit** | `edit INDEX [n\NAME] [ic\IC_NUMBER] [dob\DATE_OF_BIRTH] [ad\ADMISSION_DATE] [w\WARD] [t\TAG]…​`<br> e.g.,`edit 2 n\James Lee w\A2`
 **Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
 **List** | `list`
```
Comment (ryanlimdx) [inline docs/UserGuide.md:305]: Similarly, can update the optional parameters here too. Thank you!

File: docs/UserGuide.md [hunk 3]
```diff
@@ -168,7 +172,7 @@ Ward: A1
 Tags:
 ```
 
-### Locating persons by name: `find`
+### Locating patients by name: `find`
```
Comment (ryanlimdx) [inline docs/UserGuide.md:175]: May need to update this to reflect ShaoWei's find update as well

**Review by ryanlimdx — COMMENTED** [review]
LGTM, just  need to reflect additional parameters based on v1.3. Otherwise can leave as v1.2's implementation for now I think.

**Review by iamtr — APPROVED** [review]
LGTM

**Pull request merged**